server/src/yml-objects/AbstractYmlFunction.ts

```diff
@@ -23,4 +23,9 @@ export abstract class AbstractYmlFunction extends AbstractYmlObject {
      * Arguments of the function.
      */
     public args: YmlArgs;
+
+    public getShortName() {
+        const functionNameSubParts = this.label.split('::');
+        return functionNameSubParts.pop();
```

Comment:
As I understand it, `pop` changes the array it is called on. Here, this incurs a tiny extra cost for nothing, right? What about…

```suggestion
        return functionNameSubParts[functionNameSubParts.length - 1];
```

?

# Edit

Oh, old code? Maybe I already yelled about it? xD